CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Changed
```

Comment:
This should be a `### Breaking Changes` as it crosses a major version and if someone has an older pinned version of http plugin they will have the reverse problem you are seeing.

Reply:
Good catch, thanks for taking the time to look!